Pipfile

```diff
@@ -15,6 +15,7 @@ flake8-quotes = "==1.0.0"
 frozendict = "==1.2"
 isort = "==4.3.4"
 mysqlclient = "==1.3.14"
+neo4j = ">=1.7.4,<2"
```

Comment:
Can we pin to exact versions? It's dev-packages, so it can't be too restrictive.

Reply:
Normally I'd agree, but this pipfile is only for the local setup and for the CI tests, and I have a gut feeling that we lose more than we gain by pinning to exact versions here. Say that 1.8 gets released and accidentally breaks something -- we'll never pick it up unless someone specifically updates the pipfile. If the pipfile is flexible then the next update to the lockfile will pick up the new version.

Ultimately, as far as I can tell, the difference doesn't seem that large one way or the other.